webviz_subsurface/containers/_inplace_volumes.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 import dash_html_components as html
```

Comment:
There are some docstrings in this file refering to `dcc.Graph`. Replace with `wcc.Graph`?